packages/office-ui-fabric-react/src/components/Shimmer/Shimmer.styles.ts

```diff
@@ -88,12 +88,12 @@ export function getStyles(props: IShimmerStyleProps): IShimmerStyles {
         left: 0,
         width: '100%',
         height: '100%',
-        background: `${shimmerColor || palette.neutralLighter}
+        background: `${shimmerColor || semanticColors.disabledBackground}
                       linear-gradient(
                         to right,
-                        ${shimmerColor || palette.neutralLighter} 0%,
-                        ${shimmerWaveColor || palette.neutralLight} 50%,
-                        ${shimmerColor || palette.neutralLighter} 100%)
+                        ${shimmerColor || semanticColors.disabledBackground} 0%,
+                        ${shimmerWaveColor || semanticColors.bodyDivider} 50%,
```

Comment:
mm, not sure we have a good color alternative for this, I wouldn't mix these slots since there's no guarantee they're different colors
could you maybe mix different opacities of disabledBackground?
otherwise I would just leave these hardcoded to palette colors until we get good semantic slots for them (better to use palette than to mix semantic slot categories, because the latter could easily break)